### PR TITLE
CfS: Session and Speaker view pages link to sessions or speakers page respectively and add spacing

### LINKED ISSUE #6469: CfS: Session and Speaker view pages link to sessions or speakers page respectively and add spacing
The Cfs session view pages and speaker view page already list the speaker or sessions respectively. 

1. Please link them to the CfS view page instead of only displaying the name of the session or speaker.
2. Increase the white space below the bottom of the form on the view page.
3. Increase the white space below the bottom on the form on the edit page of sessions and speakers.

Screenshot from 2021-01-23 19-59-16

Screenshot from 2021-01-23 19-58-47

### PR DESCRIPTION
**Pull request by akash2408**
Fixes #6469 

#### Short description of what this resolves:
add marigin bottom of 4
space 1

add link to view-session and view-speakers
new

from speaker view
first
 
from organizer view
second

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [ ] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [ ] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/controllers/public/cfs/view-session.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default class extends Controller.extend(FormMixin) {
 
   @computed('model.session')
   get allSpeakers() {
-    return this.model.session.get('speakers').map(speaker => speaker.name).join(', ');
+    return this.model.session.get('speakers');
```
Comment (iamareebjamal) [inline app/controllers/public/cfs/view-session.js:20]: Useless

File: app/controllers/public/cfs/view-speaker.js [hunk 2]
```diff
@@ -17,6 +17,6 @@ export default class extends Controller.extend(FormMixin) {
 
   @computed('model.session')
   get allSessions() {
-    return this.model.speaker.get('sessions').map(session => session.title).join(', ');
+    return this.model.speaker.get('sessions');
```
Comment (iamareebjamal) [inline app/controllers/public/cfs/view-speaker.js:20]: Useless

**Comment by iamareebjamal** [comment]
Show them horizontally

**Comment by akash2408** [comment]
> Show them horizontally

ohk and i should use comma for sepration

**Comment by mariobehling** [comment]
Thanks. Please also take care of the following:
1. Show each session or speak on a new line
2. If a session has two speakers only link to speaker profiles that the speaker is permitted to edit, e.g. speakers without other roles in the system can only edit their own details (but organizers can edit details of all speakers. So if an organizer also submits a session they would be able to edit all speakers here. This should be role specific therefore.)

File: app/routes/public/cfs/view-speaker.js [hunk 3]
```diff
@@ -28,10 +28,4 @@ export default class EditSessionRoute extends Route {
     };
   }
 
-  afterModel(model) {
-    super.afterModel(...arguments);
-    if (!(this.session.isAuthenticated && (this.authManager.currentUser.isAdmin || this.authManager.currentUser.get('email') === model.speaker.get('email')))) {
-      this.transitionTo('not-found');
-    }
-  }
```
Comment (maze-runnar) [inline app/routes/public/cfs/view-speaker.js:36]: don't remove this

**Review by maze-runnar — DISMISSED** [review]

File: app/routes/public/cfs/view-speaker.js (hunk 3 above)
Comment (maze-runnar) [reply app/routes/public/cfs/view-speaker.js:36]: @MeghalBisht please do the changes, if @akash2408 is not available.

**Review by MeghalBisht — APPROVED** [review]

**Review by maze-runnar — APPROVED** [review]

**Pull request merged**